meta-client/src/main/java/org/apache/helix/metaclient/constants/MetaClientConstants.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.helix.metaclient.constants;
+
+public interface MetaClientConstants {
```

Comment:
Also why do we make this class an interface?

Reply:
I followed the current Helix constants class and uses interface class. It seems to be an anti pattern behavior after some research. 
Changed to a final class with private constructor.